### PR TITLE
Changed properties loader to MessageSource

### PR DESCRIPTION
**Pull request by Garuda-1**
MessageSource provides out-of-the-box formatting and l10n. Besides, it leads to
a significant code simplification.

### CODE REVIEW

File: src/main/java/ru/itmo/sd/deadliner2bot/state/AddTodoNotificationState.java [hunk 1]
```diff
@@ -28,13 +26,13 @@
...
     private final ChatRepository chatRepository;
-    private final MessageUtils messageUtils;
+    private final MessageSourceUtils messageSourceUtils;
     private final DateTimeUtils dateTimeUtils;
     private final TodoService todoService;
     private final TodoNotificationService todoNotificationService;
     private final Commands commands;
-    private final StateMessages stateMessages;
-    private Map<String, CommandInfo> commandsInfo;
+    //    private final StateMessages stateMessages;
```
Comment (Deletrius366) [inline src/main/java/ru/itmo/sd/deadliner2bot/state/AddTodoNotificationState.java]: remove?

File: src/main/java/ru/itmo/sd/deadliner2bot/ui/messages/MessageFormatter.java [hunk 2]
```diff
@@ -1,83 +1,112 @@
...
+
+    public BotApiMethod<Message> stateHelpMessage(Chat chat, ChatStateEnum chatStateEnum) {
+        List<String> commands = messageSource
+                .getCommonPropertiesCodes(c -> c.startsWith(chatStateEnum + ".") && c.endsWith(".cmd"))
+                .stream()
+                .sorted()
+                .map(messageSourceUtils::getCommonProperty)
+                .map(this::escape)
+                .collect(Collectors.toCollection(ArrayList::new));
+        List<String> descriptions = messageSource
```
Comment (Deletrius366) [inline src/main/java/ru/itmo/sd/deadliner2bot/ui/messages/MessageFormatter.java]: maybe factor out common code, looks like copy-paste

File: src/main/java/ru/itmo/sd/deadliner2bot/state/AddTodoNotificationState.java (hunk 1 above)
Comment (Garuda-1) [reply src/main/java/ru/itmo/sd/deadliner2bot/state/AddTodoNotificationState.java]: Good finding

File: src/main/java/ru/itmo/sd/deadliner2bot/ui/messages/MessageFormatter.java (hunk 2 above)
Comment (Garuda-1) [reply src/main/java/ru/itmo/sd/deadliner2bot/ui/messages/MessageFormatter.java]: Done

**Review by Deletrius366 — APPROVED** [review]

**Pull request merged**